django/conf/__init__.py

```diff
@@ -198,7 +200,7 @@ def __init__(self, default_settings):
         self.default_settings = default_settings
 
     def __getattr__(self, name):
-        if name in self._deleted:
+        if not name.isupper() or name in self._deleted:
```

Comment:
I'm not sure about this check. There were some past performance concerns about additional logic in `__getattr__()` at https://github.com/django/django/pull/9974/files#r190244906. The issue doesn't seem serious enough to warrant any performance regressions so perhaps it would be better to skip this validation unless it can be done another way.

Reply:
`LazySettings.__getattr__()` caches every result so performance shouldn't be an issue. A worst-case scenario where it's running on a low-end machine and accessing 500 settings would only add ~100usec of total runtime. Compared to the ~1.25msec of time it takes to access the actual settings, it's a drop in the bucket.